pyomo/contrib/trustregion/PyomoInterface.py

```diff
@@ -542,9 +540,10 @@ def linConMaker(l, i):
         gfnorm = sqrt(sum(x[1]**2 for x in g))
 
 
-        opt = SolverFactory(self.solver, solver_io=self.solver_io)
-        opt.options['halt_on_ampl_error'] = 'yes'
-        opt.options['max_iter'] = 5000
+        opt = SolverFactory(self.config.solver)
+        opt.options.update(self.config.solver_options)
+        #opt.options['halt_on_ampl_error'] = 'yes'
+        #opt.options['max_iter'] = 5000
```

Comment:
Can these commented lines be removed?

Reply:
Yes, although they were hard-coded options that @johneason originally put in.  I was going to leave them in for a while in case we wanted to port them over as defaults in the future.

Reply:
That's fine, just wanted to check.